sql/mob_skill_lists.sql

```diff
@@ -3450,3 +3450,14 @@ INSERT INTO `mob_skill_lists` VALUES ('Ignamoth',1006,1952);
 INSERT INTO `mob_skill_lists` VALUES ('Ignamoth',1006,1956);
 INSERT INTO `mob_skill_lists` VALUES ('Ankabut',1007,704);
 INSERT INTO `mob_skill_lists` VALUES ('White_Coney',1008,323); -- only Wild Carrot
+INSERT INTO `mob_skill_lists` VALUES ('Ophiotaurus',1009,2925); -- Bonebreaking Barrage
+INSERT INTO `mob_skill_lists` VALUES ('Ophiotaurus',1009,2923); -- Calcifying Claw
+INSERT INTO `mob_skill_lists` VALUES ('Ophiotaurus',1009,2924); -- Divesting Stampede
+INSERT INTO `mob_skill_lists` VALUES ('Ophiotaurus',1009,3506); -- Hellfire Arrow
+INSERT INTO `mob_skill_lists` VALUES ('Ophiotaurus',1009,3507); -- Incensed Pummel
+INSERT INTO `mob_skill_lists` VALUES ('Ophiotaurus',1009,2922); -- Soulshattering Roar
+INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3511); -- Lunatic Voice
+INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3514); -- Hysteric Assault
+INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3513); -- Entice
+INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3512); -- Sonic Buffet
```

Comment:
Hate to sound like a nag, but could these be sorted by List ID -> Skill ID~?

(Seems to be the convention established elsewhere in the file.)